documentation/modules/ROOT/nav.adoc

```diff
@@ -11,6 +11,7 @@
 ** xref:configuration/mongodb-event-flattening.adoc[MongoDB New Document State Extraction]
 ** xref:configuration/outbox-event-router.adoc[Outbox Event Router]
 ** xref:configuration/serdes.adoc[Change Event SerDes]
+** xref:configuration/cloudevents.adoc[CloudEvents]
```

Comment:
Would it make more sense for this to be under the integration section with SerDes and Outbox?

Reply:
Yes, good idea. Will do.